internal/datasystem/data_status.go

```diff
@@ -0,0 +1,12 @@
+package datasystem
+
+type DataAvailability string
+
+const (
+	// Defaults means the SDK has no data and will evaluate flags using the application-provided default values.
```

Comment:
Maybe this should be a function from timestamp to status.

So a given timestamp is Default, Stale, Fresh.